### PR TITLE
[dontmergeyet] jenkins: Use develcloud6 as default for cct PRs

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

**Comment by toabctl** [comment]
FYI: just a sync done via japi...

**Comment by dirkmueller** [comment]
LGTM. I wonder if we shouldn't switch the default to develcloud6 globally though..

**Comment by toabctl** [comment]
> LGTM. I wonder if we shouldn't switch the default to develcloud6 globally though..

+1 for switching the default. @bmwiedemann , @jdsn any objections?

**Comment by rsalevsky** [comment]
We already talked in the stand up about this our idea was to test both.

**Comment by toabctl** [comment]
> We already talked in the stand up about this our idea was to test both.

For cct, we don't need that. There's a cloud5 branch. For automation I thought we have currently not enough power to test both

**Comment by rsalevsky** [comment]
Yeah, you are right but I can't remember what was our conclusion at the end. But back to topic, i'm fine with this change. :+1:

**Comment by dirkmueller** [comment]
hmm, now that I think about it, I'm not sure what you're trying to fix here.. there is no PR testing implemented for cct yet. (it always checks out unmodified branch, not branch+PR). -1

**Comment by toabctl** [comment]
Ah. I tried to fix the case that a PR for cct was always testing develcloud5 .  But good to know that the branch is not used. I already wondered why the added tests from https://github.com/SUSE-Cloud/cct/pull/24 are not executed during a testrun...

File: scripts/jenkins/ci.suse.de/cloud-automation-pr-trigger.xml [hunk 1]
```diff
@@ -81,7 +81,12 @@ for repo in ${cloud_repos[@]}; do
         if [ "$github_pr_branch" = "master" ]; then
             # default config / may be overridden by tags from PR message
             mkcloudtarget=all_noreboot
-            cloudsource=develcloud5
```
Comment (jdsn) [inline scripts/jenkins/ci.suse.de/cloud-automation-pr-trigger.xml]: Shorter and safer:
```
cloudsource=develcloud5
# cct has cloud version related branches so default to latest version
if [ "$repo" = "SUSE-Cloud/cct" ]; then
    cloudsource=develcloud6
fi
```

**Pull request closed without merging**